### ISSUE TITLE
Coding exon limit

### ISSUE THREAD

**Issue by husamia** [lh3/miniprot#1]
Does it work will with super short exons like a single codon in length?

**Comment by lh3** [comment]
No, it doesn't. Small exons are hard. There are multiple easier and more obvious issues I need to improve first.
